setup.py

```diff
@@ -24,7 +24,7 @@
 # Requirements should be as minimal as possible.
 # Avoid pinning, and use minimum version numbers
 # only where required.
-requirements = ["pennylane>=0.16", "cirq>=0.9"]
+requirements = ["pennylane>=0.16", "cirq>=0.9,<0.12"]

```

Comment:
Strange that we're pinning `cirq>=0.9` here, but `cirq>=0.10` in the `requirements.txt` :thinking: